app/bundles/CoreBundle/Form/DataTransformer/SortableListTransformer.php

```diff
@@ -41,7 +44,12 @@ public function reverseTransform($array)
         return $this->formatList($array);
     }
 
-    private function formatList($array)
+    /**
+     * @param array<mixed>|null $array
+     *
+     * @return array<mixed>
+     */
+    private function formatList(?array $array): array
```

Comment:
I started a discussion about changing method signature and if we should still consider it a BC break:

https://mautic.slack.com/archives/CQMKV0RU1/p1704705746283329

Reply:
I missed that this is a private method. Not a BC break